### PR TITLE
zh-cn updates

### PR DESCRIPTION
**Pull request by MarcaWitt**

### CODE REVIEW

File: blog/zh-cn/architecture-design.md [hunk 1]
```diff
@@ -45,9 +45,9 @@
 
 #### 2.2 架构说明
 
-* **MasterServer** 
+* **MasterServer**
 
-    MasterServer采用分布式无中心设计理念，MasterServer主要负责 DAG 任务切分、任务提交监控，并同时监听其它MasterServer和WorkerServer的健康状态。
+    MasterServer采用分布式无中心设计理念并且可以实现功能化，MasterServer主要负责 DAG 任务切分、任务提交监控，并同时监听其它MasterServer和WorkerServer的健康状态，直接通信进行任务分配，降低。
```
Comment (CalvinKirs) [inline blog/zh-cn/architecture-design.md:50]: It still sounds ambiguous, and the expression is not clear enough

Comment (choba731) [reply blog/zh-cn/architecture-design.md:50]: whether the "MasterServer采用应用无中心设计理念并且功能化，MasterServer主要负责DAG任务切分、提交监控，监听其他MasterServer和WorkerServer的健康状态，直接通信进行任务分配，降低。“ statement is reasonable?
@CalvinKirs

Comment (choba731) [reply blog/zh-cn/architecture-design.md:50]: resolved @CalvinKirs

**Comment by dailidong** [comment]
@MarcaWitt , I noticed the status of this PR is in draft, do you have any update since last commit ?

image

**Comment by MarcaWitt** [comment]
@dailidong  I've been trying to perfect, but no one's sustainable

**Comment by MarcaWitt** [comment]
@dailidong I've been trying to perfect, but no one's continuous guidance.

File: blog/zh-cn/architecture-design.md [hunk 2]
```diff
@@ -60,47 +59,45 @@
 
     - **MasterTaskExecThread**主要负责任务的持久化
 
-* **WorkerServer** 
+* **WorkerServer**
 
-     WorkerServer也采用分布式无中心设计理念，WorkerServer主要负责任务的执行和提供日志服务。WorkerServer服务启动时向Zookeeper注册临时节点，并维持心跳。
+     WorkerServer采用无中心设计，Worker去数据库操作，只负责任务运行，职责更加单一。
+     Worker节点负责均衡策略：随机、循环及cpu和内存的线性加权负载均衡。
+
```
Comment (CalvinKirs) [inline blog/zh-cn/architecture-design.md:66]: typo～

File: blog/zh-cn/architecture-design.md [hunk 3]
```diff
@@ -45,10 +45,9 @@
 
 #### 2.2 架构说明
 
-* **MasterServer** 
+* **MasterServer**
 
-    MasterServer采用分布式无中心设计理念，MasterServer主要负责 DAG 任务切分、任务提交监控，并同时监听其它MasterServer和WorkerServer的健康状态。
-    MasterServer服务启动时向Zookeeper注册临时节点，通过监听Zookeeper临时节点变化来进行容错处理。
+    MasterServer采用无中心设计，Master多种策略分发任务，Master和Worker直接通信进行任务分配，降低延时。
```
Comment (CalvinKirs) [inline blog/zh-cn/architecture-design.md:50]: Reducing latency is not the goal, nor the result.

**Comment by dailidong** [comment]
sorry, you can ask me, my wechat: taskflow, I think you can do it better

**Pull request closed without merging**